### ISSUE TITLE
would be nice if you could add 1.20.2 support :)

### ISSUE THREAD

**Issue by CloudeLecaw** [aematsubara/Roulette#31]
currently roulette throws an erro on loading in the new spigot 1.20.2

```
[10:00:31] [Server thread/INFO]: [Roulette] Enabling Roulette v1.9.6
[10:00:31] [Server thread/WARN]: java.lang.NoSuchMethodException: no such method: net.minecraft.server.network.PlayerConnection.a(Packet)void/invokeVirtual
[10:00:31] [Server thread/WARN]: 	at java.base/java.lang.invoke.MemberName.makeAccessException(MemberName.java:915)
[10:00:31] [Server thread/WARN]: 	at java.base/java.lang.invoke.MemberName$Factory.resolveOrFail(MemberName.java:994)
[10:00:31] [Server thread/WARN]: 	at java.base/java.lang.invoke.MethodHandles$Lookup.resolveOrFail(MethodHandles.java:3663)
[10:00:31] [Server thread/WARN]: 	at java.base/java.lang.invoke.MethodHandles$Lookup.findVirtual(MethodHandles.java:2689)
[10:00:31] [Server thread/WARN]: 	at me.matsubara.roulette.util.xseries.ReflectionUtils.<clinit>(ReflectionUtils.java:240)
[10:00:31] [Server thread/WARN]: 	at me.matsubara.roulette.RoulettePlugin.onEnable(RoulettePlugin.java:76)
[10:00:31] [Server thread/WARN]: 	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:266)
[10:00:31] [Server thread/WARN]: 	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:342)
[10:00:31] [Server thread/WARN]: 	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:480)
[10:00:31] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_20_R2.CraftServer.enablePlugin(CraftServer.java:548)
[10:00:31] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_20_R2.CraftServer.enablePlugins(CraftServer.java:462)
[10:00:31] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:577)
[10:00:31] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:400)
[10:00:31] [Server thread/WARN]: 	at net.minecraft.server.dedicated.DedicatedServer.e(DedicatedServer.java:250)
[10:00:31] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:954)
[10:00:31] [Server thread/WARN]: 	at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:298)
[10:00:31] [Server thread/WARN]: 	at java.base/java.lang.Thread.run(Thread.java:1623)
[10:00:31] [Server thread/WARN]: Caused by: java.lang.NoSuchMethodError: 'void net.minecraft.server.network.PlayerConnection.a(net.minecraft.network.protocol.Packet)'
[10:00:31] [Server thread/WARN]: 	at java.base/java.lang.invoke.MethodHandleNatives.resolve(Native Method)
[10:00:31] [Server thread/WARN]: 	at java.base/java.lang.invoke.MemberName$Factory.resolve(MemberName.java:962)
[10:00:31] [Server thread/WARN]: 	at java.base/java.lang.invoke.MemberName$Factory.resolveOrFail(MemberName.java:991)
[10:00:31] [Server thread/WARN]: 	... 15 more
```

**Comment by aematsubara** [comment]
Yes I'm aware of this issue, I'm waiting for some dependencies to update first!

**Comment by CloudeLecaw** [comment]
if this is protocollib... there was a 669 build released for 1.20.2 on jenkins

**Comment by aematsubara** [comment]
> if this is protocollib... there was a 669 build released for 1.20.2 on jenkins

That was XSeries, Its updated now; I'll make an update in these days, I'm adding new features too!

**Comment by aematsubara** [comment]
Updated!
